src/HW3.py

```diff
@@ -0,0 +1,58 @@
+a = " I love Python"
+b = a*42
```

Comment:
nit: use spaces before and after operator
```suggestion
b = a * 42
```